plugins/jira-dashboard/src/components/JiraDashboardContent/JiraDashboardContent.tsx

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  Content,
+  ContentHeader,
+  Progress,
+  ResponseErrorPanel,
+  SupportButton,
+} from '@backstage/core-components';
+import { Grid } from '@material-ui/core';
+import React from 'react';
+import { JiraProjectCard } from '../JiraProjectCard';
+import { JiraTable } from '../JiraTable';
+import { useApi } from '@backstage/core-plugin-api';
+import { useEntity } from '@backstage/plugin-catalog-react';
+import { PROJECT_KEY_ANNOTATION } from '../../annotations';
+import { stringifyEntityRef } from '@backstage/catalog-model';
+import { jiraDashboardApiRef } from '../../api';
+import { useJira } from '../../hooks/useJira';
+import { JiraDataResponse } from '../../types';
+
+export const JiraDashboardContent = () => {
+  const { entity } = useEntity();
+  const projectKey = entity?.metadata.annotations?.[PROJECT_KEY_ANNOTATION]!;
```

Comment:
Optional: If the `projectKey` is undefined would it make sense to add the missing annotation panel?

Reply:
Good comment! My plan is to look into more user friendly error handling in the next PR :)

Reply:
That works for me, maybe bookmark this then: https://backstage.io/storybook/?path=/story/feedback-emptystate--missing-annotation. That's the component you'll want to use when the annotation is missing